static/app/views/performance/data.tsx

```diff
@@ -786,6 +786,7 @@ export function generatePerformanceVitalDetailView(location: Location): EventVie
       getVitalDetailTableMehStatusFunction(vitalName),
     ],
     version: 2,
+    yAxis: [`p75(${vitalName})`],
```

Comment:
Was `p75` a deliberate choice here? I don't really have too much context on this, so I'm curious why we're defaulting to p75 here

Reply:
@udameli 

I chose p75 since it appears to be the default used by the chart on the Vital Detail page:
![image](https://user-images.githubusercontent.com/16740047/165989782-db54cf69-d37e-4a39-ab62-2d3c06cc0d34.png)

Reply:
ah gotcha 👍